pkg/api/health.go

```diff
@@ -208,23 +209,20 @@ func PrintOverallReleaseHealthFromDB(w http.ResponseWriter, dbc *db.DB, release
 	}
 	currStats, prevStats := calculateJobResultStatistics(jobReports)
 
+	warnings := make([]string, 0)
+	releaseWarnings, err := ScanReleaseHealth(dbc, release)
+	if err != nil {
+		warnings = append(warnings, fmt.Sprintf("error checking release health, see logs: %v", err))
+	} else {
+		warnings = append(warnings, releaseWarnings...)
+	}
```

Comment:
If we plan to have more warnings, it might be nice to extract this to it's own function so we don't start inserting a bunch of warning generation inline